LPS-205243

Mikellorza opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

veroglez committed:
LPS-202929 Add data-language-id and data-field-name to the hidden inputs to have better access to the languageid

veroglez committed:
LPS-202928 Update the field names

veroglez committed:
LPS-202928 Rebuild the structure of locales and rename the method

veroglez committed:
LPS-202928 Add translationProgress prop to show how many translated fields there are at the moment

veroglez committed:
LPS-202928 Update translation manager with TranslationAdminSelector component

veroglez committed:
LPS-202929 Update the translation when the event from localized input is listened

izaera committed:
LPS-204464 Remove FF for CSPs nonces

Commit:
LPS-205232 restore css bulider version

Commit:
LPS-202063 Using ConfigurationProvider to tests

adolfopa committed:
LPS-199506 Property to specify a "blocked" folder

A blocked folder is a folder where navigation is not possible. This is
to prevent operations like copying a folder inside itself (or one of
its descendants). Preventing navigation will make this impossible from
the UI, while the operation will fail gracefully if performed by other
means.

adolfopa committed:
LPS-199506 Let each operation provide its own URL

Different targets will need to use different URLs. Let each one pass
the correct URL to the Js method.

Commit:
LPS-202063 Fix integration tests after using ConfigurationProvider

Commit:
LPS-202063 Adding new scenario

adolfopa committed:
LPS-199506 Consolidate all folder selection urls in one util class

adolfopa committed:
LPS-199506 Copy and move criteria should be the same

There's no reason for these to be different. Pick the more explicit
one.

adolfopa committed:
LPS-199506 SF - rename methods

adolfopa committed:
LPS-199506 Provide selection URL to frontend side

adolfopa committed:
LPS-199506 When moving/copying a folder, mark it as blocked

adolfopa committed:
LPS-199506 Make class attribute final

adolfopa committed:
LPS-199506 Start in the parent folder, and prevent navigation to selected folder

adolfopa committed:
LPS-204273 LPS-204274 LPS-204276 Remove unused references from DLApp wrappers

adolfopa committed:
LPS-203653 Remove generic type from SF rules

igor-franca committed:
LPS-197643 Increase Handle Size and Create a Connection Area Around the Node

carolmariaabb committed:
LPS-204746 Move logic to the API to be reused

carolmariaabb committed:
LPS-204746 Add DDMFormFields to DDMFormInstanceRecordWriterRequest

carolmariaabb committed:
LPS-204746 baseline

Commit:
LPS-204749 Fix message from exception

carolmariaabb committed:
LPS-204746 Improve data format for numeric and date fields when exporting form entries

Commit:
LPS-204749 Adding validation for some fields on ObjectConfiguration

carolmariaabb committed:
LPS-204746 Fix tests

Commit:
LPS-202063 Fix test

gabrielwas committed:
LPS-204716 Split into methods

gabrielwas committed:
LPS-204716 Include portalURL in the general variables

Commit:
LPS-204749 Adding text to language.properties

AliciaGarciaGarcia committed:
LPS-199506 baseline

AliciaGarciaGarcia committed:
LPS-199506 autoSF

sergiojimcos committed:
LPS-201696 Add status property to ObjectDefintionEntityModel and ObjectDefinitionModelDocumentContributor to filter by status

sergiojimcos committed:
LPS-201696 Add test case filtering by status code

sergiojimcos committed:
LPS-201696 Apply new filter to show published Object Definitions in API Schema dropdown

gabrielwas committed:
LPS-204716 Create constants for notification freemarker template

gabrielwas committed:
LPS-204716 Replace portalURL variable by the current value of portalURL when building the notification

victorg1991 committed:
LPS-204925 Update debounce typing

Commit:
LPS-204749 Fix on message and test

Commit:
LPS-204749 Fix error message on forms

Commit:
LPS-204749 Build Lang

magjed4289 committed:
LPS-204874 Adapting test cases to current validation in the Object layer

Commit:
LPS-205096 - Update a11y-sample-web to ESM

qz1010 committed:
LPS-204967 fix logic, only allow skipping certain directories

sergiojimcos committed:
LPS-201696 Update Poshi tests

Commit:
LPS-205096 - Update a11y-web to ESM

Commit:
LPS-205096 - Update alert-support-web to ESM

magjed4289 committed:
LPS-204874 Object name must begin with upper case

Commit:
LPS-205096 - Update clay-sample-web to ESM

Mikellorza committed:
LPS-204982 Add friendly url resolver separator company configuration

Commit:
LPS-205096 - Update collapse-support-web to ESM

Commit:
LPS-205096 - Update walkthrough-web to ESM

Commit:
LPS-205096 - Update components-sample-web to ESM

AliciaGarciaGarcia committed:
LPS-157775 extract methods of upgrade dl configurations processes to reuse several times

AliciaGarciaGarcia committed:
LPS-157775 add new step because sometimes the configuration is not correctly upgraded, so we must redo it

AliciaGarciaGarcia committed:
LPS-157775 only do the upgrade if there was not user changes on the new properties or the old ones are still with data

AliciaGarciaGarcia committed:
LPS-157775 semver

AliciaGarciaGarcia committed:
LPS-157775 extract methods to clarification

Mikellorza committed:
LPS-204982 Add friendly url resolver separator manager to manage company configurations

Mikellorza committed:
LPS-204982 Add friendly url separator configuration custom screen

gabrielwas committed:
LPS-204716 baseline

magjed4289 committed:
LPS-204874 Requested changes

magjed4289 committed:
LPS-204874 Retrieve and modify incorrectly removed test case

Mikellorza committed:
LPS-204982 Add getKey to identify the friendly url resolver, add isURLSeparatorConfigurable to know if the url separator is configurable or not and add getDefaultURLSeparator to get the default url separator if it is not configured

Mikellorza committed:
LPS-204982 Add action command to save friendly url company configuration

Commit:
LPS-205096 - Update dropdown-support-web to ESM

Commit:
LPS-205096 - Update spa-web to ESM

Mikellorza committed:
LPS-204982 Add info for the friendly url separator company configuration view

Mikellorza committed:
LPS-204982 Add language keys

Mikellorza committed:
LPS-204982 Show friendly url separators in the config

AliciaGarciaGarcia committed:
LPS-199506 as the _blockedFolderId default value is (DLFolderConstants.DEFAULT_PARENT_FOLDER_ID ignore if it is it

Commit:
LPS-205096 - Update tabs-support-web to ESM

Commit:
LPS-205096 - Update tooltip-support-web to ESM

Commit:
LPS-205096 - Update walkthrough-sample-web to ESM

AliciaGarciaGarcia committed:
LPS-199506 now info panel needs a itemSelector on request to avoid NPE

Commit:
LPS-205096 - Remove this file since it isn't even used

magjed4289 committed:
LPS-204874 Sort

ZakiMiklos committed:
LPS-146160 use correct field to check if template stageing is allowed

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/model/LayoutSet.java

The method "public boolean isLayoutSetReadyForPropagation()" has been created

## Why

The new method can identify whether the LayoutSet is actively propagated from a LayoutSetPrototype.

----

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/model/LayoutSetWrapper.java

The method "public boolean isLayoutSetReadyForPropagation()" has been created

## Why

The new method can identify whether the LayoutSet is actively propagated from a LayoutSetPrototype.

----

AliciaGarciaGarcia committed:
LPS-157775 move to helper

AliciaGarciaGarcia committed:
LPS-157775 split on several classes to make it clear that we are redoing the upgraded processes again

guilhermedcamacho committed:
LPS-202063 Merge per day tests

guilhermedcamacho committed:
LPS-202063 Just one test per week is enough

guilhermedcamacho committed:
LPS-202063 Improve test

danvb112 committed:
LPS-204046 Add selected object folder name to the URL to correctly redirect the user back from Model Builder to the folder listing

tiago-ds committed:
ISSD-2246 Update style books on Learn Project

ling-alan-huang committed:
LPS-204967 SF

ling-alan-huang committed:
LPS-204967 Rename

austinchiang committed:
Revert "LPS-204273 LPS-204274 LPS-204276 Remove unused references from DLApp wrappers"

This reverts commit 979ee394b861ebedf45489625d24abe4ae4d1c40.

Commit:
LPS-205096 - SF

lfbesada committed:
Revert "LPS-203513 Disable input when configuring non-default Display Pages"

This reverts commit a3dfbc49e3d68c4c29e7083ffbe44da5fcf69869.

lfbesada committed:
LPS-204884 Removes UI restrictions on robots and include-in-sitemap fields on the configuration of NON site default DPT

lfbesada committed:
LPS-204884 Removes unused language keys

lfbesada committed:
LPS-204884 BuildLang

Mikellorza committed:
LPS-204982 SF

Mikellorza committed:
LPS-204982 Add FF LPS-203351

Commit:
LPS-205096 - Update node-shims to ESM

lfbesada committed:
LPS-200303 Don't apply the workflow permissions for the VIEW's action on published layouts as the actually workflowed layout is the draft one so in this case the published layout VIEW model permission should be applied

lfbesada committed:
LPS-200303 Adds integration tests to assert that the Guest user is allowed to view a published content page or not depending on the layout VIEW permissions for the Guest users

lfbesada committed:
LPS-200303 Adds integration tests to assert that the Guest user is allowed to view the published content page or not depending on the layout VIEW permissions for the Guest users when there is a new page version at pending status

lfbesada committed:
LPS-200303 Adds integration tests to assert that the Guest user is NOT allowed to view a never published content page when is in pending status even if the Guest user has VIEW permissions over the layout

sergiojimcos committed:
LPS-201696 Add Indexable annotation in publishCustomObjectDefinition and publishSystemObjectDefinition

sergiojimcos committed:
LPS-201696 buildService

BarbaraCabrera committed:
LPS-205102 Add PermissionsPortletConfigurationIcon to show the permissions option in the kebab menu and modify the weight to adapt the order

AliciaGarciaGarcia committed:
LPS-157775 not needed

AliciaGarciaGarcia committed:
LPS-157775 split method

AliciaGarciaGarcia committed:
LPS-157775 rename

AliciaGarciaGarcia committed:
LPS-157775 rename and refactor

LuisOrtiz89 committed:
LPS-205216 Missing argument for formatting expression

adolfopa committed:
LPS-199506 Initialize item selector in portlet class

AliciaGarciaGarcia committed:
LPS-193971 extract before shorten because short first can cause the deletion of the end of a html tag

lfbesada committed:
LPS-205255 SF removes as is redundant

lfbesada committed:
LPS-205255 Take into account the utility pages

lfbesada committed:
LPS-205255 SF buy space

carolmariaabb committed:
LPS-194017 Also search for nested fields when propagating fieldSet changes

ealonso committed:
LPS-205267 Adds new render command for edit journal article

lfbesada committed:
LPS-205255 Adds required dependency

lfbesada committed:
LPS-205255 Provides a specific display context for the UtilityPages

ealonso committed:
LPS-205267 Uses portlet builder

lfbesada committed:
LPS-205255 Overrides isWorkflowEnabled to disabled it for utility pages

lfbesada committed:
LPS-205255 Adds required dependency

ealonso committed:
LPS-205267 Uses new render command

ealonso committed:
LPS-205267 Creates JournalEditArticleDisplayContext only once

ealonso committed:
LPS-205267 Uses class method

ealonso committed:
LPS-205267 Inline method

AliciaGarciaGarcia committed:
LPS-205121 Servlet should check the upload servlet maxsize

ealonso committed:
LPS-205267 Uses local service and fetch, since we don't need to check permissions for this case

ealonso committed:
LPS-205267 Reuse method

adolfopa committed:
LPS-205263 Change url separator to avoid conflicts

'g' is used by default for product categories (see
CPFriendlyURLConfiguration).

adolfopa committed:
LPS-205263 Semver

MarinhoFeliphe committed:
LPS-204716 Rename to _freeMarkerTermValues

sandrodw3 committed:
LPS-204982 Add required files for rendering react and taglibs

tiago-ds committed:
ISSD-2246 Rename CSS Variables

sandrodw3 committed:
LPS-204982 Add SeparatorFields component

tiago-ds committed:
ISSD-2220 Adjust Commerce capability's localizable description according to spreadsheet

tiago-ds committed:
ISSD-2246 add stylebook variables according to CSS

tiago-ds committed:
ISSD-1638 Update product cards style

MarinhoFeliphe committed:
LPS-204716 Cover portalURL FTL variable

MarinhoFeliphe committed:
LPS-204716 Isolate this case in another test because it was affected by https://github.com/liferay-echo/liferay-portal/pull/14829/commits/df6b8866344e716d6fc7f3074839b96e1f47540f

MarinhoFeliphe committed:
LPS-204716 It is only used in one place

interaminense committed:
LRAC-0 Fixing unit tests

larissacribeiro committed:
LPS-197751 Update method and it's uses with new parameter and validation"

larissacribeiro committed:
LPS-197751 buildService

larissacribeiro committed:
LPS-197751 Disable read only when field is required

fmarcio committed:
LRAC-15330 update validation to make only authorized members to download all requests from the request log

fmarcio committed:
LRAC-15330 update addParam func to check if there is a defined value

fmarcio committed:
LRAC-15330 update unit test

pedroliveirav committed:
ISSD-2015 Create support ticket escalation structure

michaelhashimoto committed:
LRCI-3954 Add method to update build description using the build url

michaelhashimoto committed:
LRCI-3954 Add methods to get find files, merge commits, & modified files in GWD

michaelhashimoto committed:
LRCI-3954 Add method to push a set of subrepository commits into portal

guilhermedcamacho committed:
LPS-204749 Sort

Commit:
LPS-204475 - Remove feature flag

sandrodw3 committed:
LPS-204982 Add jsp and render SeparatorFields component

sandrodw3 committed:
LPS-204982 Types

vendeltoreki committed:
LPS-146160 Should still work with private pages

slnn committed:
LPS-205097 Make sure the guest users are generated before generating UserPersonalSiteGroupModel as the it has a dependeny on the Gues userId in java code

slnn committed:
LPS-205097 Remove left over guest_user.ftl as we just need to generate guest users for the default company

joseabelenda committed:
ISSD-2246 Typo

joseabelenda committed:
ISSD-2246 Remove blank lines

joseabelenda committed:
ISSD-2246 Auto SF

joseabelenda committed:
ISSD-2246 Add missing name_18n

victorg1991 committed:
LPS-205116 Adjust sidebar sizing

Commit:
LPS-205373 - Add missing dependency, aui-component.

joseabelenda committed:
ISSD-2246 No longer necesary

abfranca committed:
ISSD-2291 Add missing variables

abfranca committed:
ISSD-2291 Apply front-end client extentions via site initializer.

abfranca committed:
ISSD-2291 Auto SF

vendeltoreki committed:
LPS-146160 Semantic versioning

MarinhoFeliphe committed:
LPS-204716 Delete notificationQueueEntry after test run and simplify recipient

vendeltoreki committed:
LPS-146160 Return false by default when the site is not created from a template

ealonso committed:
LPS-205319 Uses an existing portlet path

DiegoHu97 committed:
LPS-205403 Do not show lock icon when it is a private layout

Mikellorza committed:
LPS-205243 Add method to count LayoutFriendlyURLs by companyId and friendlyURL start with the given friendlyURL or exact match

veroglez committed:
LPS-205235 Add new param languagesDropdownVisible to hide the language selector

veroglez committed:
LPS-205235 Hide the language selector for Description and Friendly URL fields

abfranca committed:
ISSD-2291 Edit css from public site navigation template

marianoalvarosaiz committed:
LPS-201832 After LPS-196393 companyId is randomly generated, so the lower id cannot be trusted. Use createDate to sort.

marianoalvarosaiz committed:
LPS-205363 After LPS-202082 portal indexes are tried to update when executing IndexUpdaterUtil#updateAllIndexes if they were not previously updated. Since the test forces the failure it will try to get updated twice.

Mikellorza committed:
LPS-205243 SF

drewbrokke committed:
LPS-205476 gradle-plugins-workspace: manually adjust up-to-date status when assemble source changes

qz1010 committed:
LPS-204967 fix properties

brookedalton committed:
LPS-205442 Update button text to match behavior

brianchandotcom committed:
LPS-203653 ran 'gw formatJavadoc'

brianchandotcom committed:
LRCI-3954 Wordsmith

brianchandotcom committed:
LRCI-3954 prep next

brianchandotcom committed:
LPS-204716 Not needed

brianchandotcom committed:
LPS-204716 SF

brianchandotcom committed:
LPS-199506 SF

shuyangzhou committed:
LPS-205919 Pull out CompanyLocalService dependency

shuyangzhou committed:
LPS-205919 Pull out Portal dependency

shuyangzhou committed:
LPS-205919 Pull out GroupLocalService dependency

Jeremy-Ch committed:
LPS-205361 Remove the redundant test

Jeremy-Ch committed:
LPS-205352 Update the macro to use the suitable ellipsis path

rachaelkoestartyo committed:
LRAC-15173 add to baseProductEntryIds

Jeremy-Ch committed:
LPS-205350 Update macro to view no filter option

Jeremy-Ch committed:
LPS-205348 Remove the unnecessary test

Mikellorza committed:
LPS-205243 buildService

victorg1991 committed:
LPS-205367 Change label

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-205557 Modify case to adapt new behavior

Mikellorza committed:
LPS-205243 Add validation to ensure that url separator is not empty, use user values in case if there are errors and add a method to get the errors if there are

adolfopa committed:
LPS-204267 Use model listener instead of service wrapper

adolfopa committed:
LPS-205542 Remove unused reference

Mikellorza committed:
LPS-205243 Validate if there is another urlSeparator with the same value

jkappler committed:
LPS-205571 Avoid changing the ERC when moving the article to and from the recycle bin

sandrodw3 committed:
LPS-204982 buildLang

jkappler committed:
LPS-205571 Adds integration test

shuyangzhou committed:
LPS-204273 Unget the wrapped service only after ServiceWrapper is unwrapped.

brianchandotcom committed:
LPS-204716 my bad

Mikellorza committed:
LPS-205243 Normalize friendly url separator to avoid contains characters like %&/()·.{}?¿/ and much more. The separator only contains - numbers(0-9) and letters(a-z lowercase but not in uppercase)

Mikellorza committed:
LPS-205243 Validate maximum and minimum size

brianchandotcom committed:
LPS-205476 prep next

Mikellorza committed:
LPS-205243 Validate that the separatorUrl does not conflict with:
1- /-/ especial friendly url separator
2-Other friendly url separator
3-Friendly url mappers
4-Language urls

brianchandotcom committed:
LPS-205476 prep next

brianchandotcom committed:
LPS-205476 prep next

brianchandotcom committed:
LPS-205476 regen

Mikellorza committed:
LPS-205243 Add language keys

Mikellorza committed:
LPS-205243 buildLang

marianoalvarosaiz committed:
LPS-202082 Use proper name. After ce6a83c4b48171279442dac5b858665dff2d7cd2 tableName string and no sql is used.

shuyangzhou committed:
LPS-204273 Simplify parameter

shuyangzhou committed:
LPS-204273 Catch up late service.

shuyangzhou committed:
LPS-204273 Semantic versioning

# breaking

## What portal-impl/src/com/liferay/portal/deploy/hot/ServiceBag.java

Added BundleContext and ServiceReference to constructor parameters.
Removed the unnecessary ClassLoader parameter which can be derived from ServiceWrapper parameter.
Removed "throws Exception" from replace() method is not thrown.

## Why

Fix improper ServiceReference ungetting.

----

Mikellorza committed:
LPS-205243 Validate if the separator is a number, since it may conflict with a layout friendlyURL because in last instance we are setting the layoutId as the friendlyURL of the layout.

michaelhashimoto committed:
LRCI-4024 Download the portal license into OSB Faro if it is set

shuyangzhou committed:
LPS-204273 Remove spring bean lookup, they all turn into osgi service anyway.

shuyangzhou committed:
LPS-205919 Pull out ConfigurationProvider dependency

shuyangzhou committed:
LPS-205919 JSONFactory and Language are portal-impl service, safe to use util.

shuyangzhou committed:
LPS-205919 Remove scr pojo service wiring design in AnalyticsCloudClient

Commit:
LPS-205905 Correct the steps

brianchandotcom committed:
LPS-204749 #sfme

brianchandotcom committed:
LPS-204749 SF

brianchandotcom committed:
LPS-204749 Wordsmith

brianchandotcom committed:
LPS-204749 Regen

brianchandotcom committed:
LPS-204749 Exception messages that show up in a server log don't end in periods unless there are more than 2 sentences.

brianchandotcom committed:
LPS-204749 Rename

brianchandotcom committed:
LPS-204273 SF

brianchandotcom committed:
LPS-205232 prep next

brianchandotcom committed:
LPS-205232 prep next

brianchandotcom committed:
LPS-205232 prep next

brianchandotcom committed:
LPS-205232 Update versions

brianchandotcom committed:
LPS-201696 SF

brianchandotcom committed:
LPS-204874 SF

shuyangzhou committed:
LPS-204273 Fix test compile

Mikellorza committed:
LPS-205243 Delete slash when return the urlSeparator to the view and set it when we save them

Mikellorza committed:
LPS-205243 Validate that the separatorUrl does not conflict with: 1-Layout with same friendlyUrl 2-Layout that contains the friendlyUrl

Mikellorza committed:
LPS-205243 If it has error not replace slashed to show the source value to the user

Mikellorza commented:
ci:open

Mikellorza commented:
ci:reopen
